conjure-java-core/src/test/java/com/palantir/conjure/java/AsyncRequestProcessingTest.java

```diff
@@ -54,9 +56,7 @@
 import java.util.OptionalInt;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+import javax.ws.rs.core.HttpHeaders;
```

Comment:
Why did we switch from guava `com.google.common.net.HttpHeaders` to javax jaxrs `javax.ws.rs.core.HttpHeaders`?

Reply:
Import bug... Will fix